src/App.js

```diff
@@ -21,8 +20,6 @@ const StyledApp = styled.div`
   min-height: 100vh;
 `;
 function App() {
-  const [cookies] = useCookies([cookieName]);
```

Comment:
Original bug is probably that you are referencing this outside of the provider (first line of render) so wont hook into updates